### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by jarmee**

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    "declaration": true,
+    "declarationDir": "dist",
+    "inlineSourceMap": true,
+    "inlineSources": true,
+    "skipLibCheck": true,
+    "emitDecoratorMetadata": false,
+    "experimentalDecorators": true,
+    "importHelpers": true,
+    "lib": ["dom", "es2018"],
+    "strict": false,
```
Comment (florianehmke) [inline tsconfig.json:19]: **Nice-to-have**
`strict: true`, maybe at a later point in time ;).

File: tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    "strictPropertyInitialization": false,
+    "noUnusedParameters": true,
+    "noUnusedLocals": true,
+    "paths": {
+      "@developerwellness/angular-typed-forms": ["./src"]
+    },
+    "typeRoots": ["./node_modules/@types/"]
+  },
+  "angularCompilerOptions": {
```
Comment (florianehmke) [inline tsconfig.json:28]: Should be the same as in the example app, specifically `fullTemplateTypeCheck` is deprecated.

**Review by florianehmke — CHANGES_REQUESTED** [review]

File: tsconfig.json (hunk 2 above)
Comment (jarmee) [reply tsconfig.json:28]: ✅

File: tsconfig.json (hunk 1 above)
Comment (jarmee) [reply tsconfig.json:19]: Yes, definitely!

**Pull request merged**